### ISSUE TITLE
`mapname` cvar is never set when joining a server, is it a bug or not?

### ISSUE THREAD

**Issue by illwieckz** [Unvanquished/Unvanquished#2381]
I'm not sure this is a bug, but `mapname` cvar is never ser when joining a server.

This cvar is only set when running a map as a server (or with a client as a server). This cvar is probably only used to make possible for the game to tell the engine what map to load while the game is entirely killed between map loads. That would mean the expected reader would be the server engine, and not clients. The client is probably not expected to use that cvar at all.

By default the value is `nomap` so that's the value seen by clients connecting to servers.

**Comment by DolceTriade** [comment]
I think if things rely on the `mapname` being set in the cgame, it is a bug. I think it's probably a good thing to set the cvar tho so we can rely on it.

**Comment by slipher** [comment]
> I think if things rely on the mapname being set in the cgame, it is a bug.

Agreed. Well, I do use it in the cgame for cg_navgenOnLoad, but that code only runs if you are hosting the game so the sgame is guaranteed to be running.
